mypyc/lib-rt/setup.py

```diff
@@ -12,7 +12,7 @@
     kwargs = {"language": "c++"}
     compile_args = []
 else:
-    kwargs = {}  # type: ignore
+    kwargs = {}  # type: ignore[var-annotated]
```

Comment:
Couldn't we just get rid of the `type: ignore` altogether?

```suggestion
    kwargs: dict[str, str] = {}
```

Reply:
A good point. I made it `dict[str, Any]` to avoid type errors.